src/olympia/addons/management/commands/process_addons.py

```diff
@@ -55,6 +57,14 @@
         'qs': [
             Q(type=amo.ADDON_PERSONA, status=amo.STATUS_PUBLIC)
         ]
+    },
+    # Run this once we've disallowed new submissions not targeting Firefox and
+    # addons.thunderbird.net is live.
+    'delete_addons_not_compatible_with_firefoxes': {
+        'method': delete_addon_not_compatible_with_firefoxes,
+        'qs': [Q(status=amo.STATUS_PUBLIC),
```

Comment:
I would have thought we'd want more than just `STATUS_PUBLIC` - probably `~Q(status=amo.STATUS_DELETED)`

Reply:
The problem is, `appsupport` is only set when `version_changed` is triggered, or when the cron to catch up is fired (once a day), but both only apply to public or nominated add-ons...

Reply:
Mmm.  But we'll be leaving a lot of Thunderbird/Seamonkey add-ons that are non-public at the time of the command being run.  Might need to filter on the `ApplicationVersion` then... can't think of any easy way to avoid weird edge cases though.

Reply:
If they are not public, I don't see a problem with that. Using `ApplicationsVersions` suffer from a similar issue, you need `current_version` to be set to check the right version...

Reply:
This seems an okay solution for the initial delete post-thundernest migration, but we still need something that scrubs _all_ the non-Firefox compatible add-ons from AMO at some point.  Though I guess we can do that in the legacy add-on bonfire?

Reply:
Yeah, that may be the solution. We'll have a similar issue, how to define what a legacy add-on is? We can't rely on filtering on `current_version`, since purely unlisted add-ons don't have one. Or will we just delete all legacy *versions* / *files* ?

Reply:
yeah.  We could scrub all the legacy versions and files and then run a clean-up of any add-ons without any versions.